src/software/SfM/main_SfMInit_ImageListing.cpp

```diff
@@ -98,6 +100,7 @@ int main(int argc, char **argv)
       << "[-g|--group_camera_model]\n"
       << "\t 0-> each view have it's own camera intrinsic parameters,\n"
       << "\t 1-> (default) view can share some camera intrinsic parameters\n"
+      << "[-u|--use_UID] Generate a UID instead of using indexes\n"
```

Comment:
Perhaps the help message must be more clear (we must mention view):
Something in the vein of: View indices : UID (unique id) or incremental numbers.